graylog2-server/src/main/java/org/graylog2/lookup/adapters/CSVFileDataAdapter.java

```diff
@@ -243,6 +244,9 @@ private Map<String, String> parseCSVFile() throws IOException {
             throw new IllegalStateException(e);
         }
 
+        if (config.isCidrLookup()) {
+            cidrLookupRef.set(cidrLookupTrie);
```

Comment:
We store the entire lookup data twice in the case that `cidrLookup` is enabled, which is probably not necessary.
Off hand I cannot find any path that will read the map from `lookupRef` in that case, which means that we shouldn't have to hang onto that memory.

Reply:
I didn't think we were actually doing anything with the `lookupRef` in this method. We create a `newLookupBuilder` but only ever add any entries to it in a `if (!config.isCidrLookup())` block. Regardless, I agree it is confusing to return a lookup value that is always empty and never used if it is a cidrLookup, so I've refactored it a bit and only set the appropriate ref.